### PR TITLE
Show mandatory fields when you select them on the portal profile

### PR DESCRIPTION
**Pull request by fdurand**
## Description

Show mandatory fields when they has been selected on the portal profile
## Impacts

No impacts
## Delete branch after merge

YES
## NEWS file entries

Enhancements
++++++++++++
- Show mandatory fields on the captive portal

### CODE REVIEW

File: html/captive-portal/templates/login.html [hunk 1]
```diff
@@ -70,8 +70,15 @@
           <div class="input">
             <span>[% i18n("Password") %]</span><input class="field" name="password" type="password" />
           </div>
+
           [% END %]
-          
+          [% FOREACH field IN mandatory_fields %]
+          <div class="input">
+            <span>[% i18n("$field") %]</span>
+            <input class="field" name="[%field%]" type="text" value="" /><br/>
```
Comment (julsemaan) [inline html/captive-portal/templates/login.html:78]: Does this mean the user registering with user/password has to fill in this form ?

Comment (fdurand) [inline html/captive-portal/templates/login.html:78]: Yes but you know what i forgot to add the code behind that, so it just appear in the form but do nothing ;-)

Comment (julsemaan) [inline html/captive-portal/templates/login.html:78]: Not sure it's worth having that though. If you're using guest + AD on the same profile then your AD users have to fill-in this form (which is not that useful)

Comment (fdurand) [inline html/captive-portal/templates/login.html:78]: Yes you are true, i just have a case with a local auth source and i need to have mandatory field email.
So i have probably have to add a checkbox in the auth source configuration to ask if we have to take care of mandatory fields on the portal profile or no.

**Pull request closed without merging**